specs/015-permission-catalog/contracts/permission-catalog.contract.ts

```
@@ -0,0 +1,74 @@
+  ROLE_DELETE: 'role.delete',
+  LEAVE_VIEW: 'leave.view',
+  LEAVE_REQUEST: 'leave.request',
+  LEAVE_APPROVE: 'leave.approve',
+  PAYROLL_VIEW: 'payroll.view',
+  PAYROLL_RUN: 'payroll.run',
+} as const;
+
+export type PermissionCode = (typeof PermissionCodes)[keyof typeof PermissionCodes];
+
+export const PlatformModules = [
+  "setting",
```

ESLint

[error] Replace `"setting"` with `'setting'`

(prettier/prettier)